app/registries/foreman/plugin.rb

```diff
@@ -317,6 +317,25 @@ def tests_to_skip(hash)
       end
     end
 
+    # Adds setting definition
+    #
+    # ===== Example
+    #
+    #   settings do
+    #     category(:cfgmgmt, N_('Configuration Management')) do
```

Comment:
I do wonder how we deal with multiple plugins attempting to define the same category and wanting different names.

Should we make it an optional argument and raise an error if the category is unknown? This allows plugins to easily add options to existing categories but also to define their own if needed.

Reply:
Current implementation has the label optional. What label should win is hard to define, so I've just decided to go with the first one as per https://github.com/theforeman/foreman/pull/8437/files#diff-876fc5b457096e3d43ca07e7fbaa3e4034dc5d2854fed8dbd60afe398e9f712cR28

Falling back to `category.constantize.humanized_category` is fine now, in the future we would probably want to fall back to name as a label.